policy/predicate/file.go

```diff
@@ -184,16 +255,36 @@ func (pred *ModifiedLines) Evaluate(ctx context.Context, prctx pull.Context) (bo
 		deletions += int64(f.Deletions)
 	}
 
-	for expr, v := range map[ComparisonExpr]int64{
-		pred.Additions: additions,
-		pred.Deletions: deletions,
-		pred.Total:     additions + deletions,
-	} {
-		if !expr.IsEmpty() && expr.Evaluate(v) {
-			return true, "", nil
+	if !pred.Additions.IsEmpty() {
+		predicateResult.Values = []string{fmt.Sprintf("+%d", additions)}
+		predicateResult.ConditionValues = []string{"added lines" + pred.Additions.String()}
+		if pred.Additions.Evaluate(additions) {
+			predicateResult.Satisfied = true
+			return &predicateResult, nil
+		}
+	}
+	if !pred.Deletions.IsEmpty() {
+		if pred.Deletions.Evaluate(deletions) {
+			predicateResult.Values = []string{fmt.Sprintf("-%d", deletions)}
+			predicateResult.ConditionValues = []string{"deleted lines" + pred.Deletions.String()}
```

Comment:
```suggestion
			predicateResult.ConditionValues = []string{"deleted lines " + pred.Deletions.String()}
```

Missing space here... could also use `fmt.Sprintf("deleted lines %s", pred.Deletions)`.